.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: node_js
 sudo: false
 node_js:
-- "6"
+- "8"
+addons:
+  chrome: stable
+env:
+  - NPM_SCRIPT=test:browserless
```

Comment:
We don't have such an npm script in Migrate. You might want to migrate that from Core if you want but for now it seems you can just remove the whole `env` block?